src/urdf-parser.cpp

```diff
@@ -144,10 +164,10 @@ namespace graphics {
       shared_ptr< ::urdf::Mesh > mesh_shared_ptr;
 
       if (visual) {
-	mesh_shared_ptr = graphics::static_pointer_cast< ::urdf::Mesh >
+	mesh_shared_ptr = urdf::static_pointer_cast< ::urdf::Mesh >
```

Comment:
This does not compile with on 14.04 with `URDFDOM_BOOST_SHARED_PTR` defined.

Reply:
Yes, I forgot that 14.04 has an urdfdom < 0.4.0. This is fixed now, and has been tested on 14.04, 16.04 & fedora 26.